src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -248,6 +253,57 @@ public ITextComponent getShortDisplayString()
         }
     }
 
+    public static class SmeltAbleOreRequest extends AbstractRequest<SmeltableOre>
+    {
+
+        private static ImmutableList<ItemStack> oreExamples;
+
+        SmeltAbleOreRequest(@NotNull final IRequester requester, @NotNull final IToken token, @NotNull final SmeltableOre requested)
+        {
+            super(requester, token, requested);
+        }
+
+        SmeltAbleOreRequest(
+                @NotNull final IRequester requester,
+                @NotNull final IToken token,
+                @NotNull final RequestState state,
+                @NotNull final SmeltableOre requested)
+        {
+            super(requester, token, state, requested);
+
+        }
+
+        @NotNull
+        @Override
+        public ITextComponent getShortDisplayString()
+        {
+            return new TextComponentTranslation(TranslationConstants.COM_MINECOLONIES_REQUESTS_SMELTABLE_ORE);
+        }
+
+        @Override
+        public List<ItemStack> getDisplayStacks()
+        {
+            if (oreExamples == null)
+            {
+                oreExamples =
+                        ImmutableList.copyOf(StreamSupport.stream(Spliterators.spliteratorUnknownSize(Item.REGISTRY.iterator(), Spliterator.ORDERED), false).flatMap(item -> {
+                            NonNullList<ItemStack> stacks = NonNullList.create();
+                            try
+                            {
+                                item.getSubItems(item,null, stacks);
+                            }
+                            catch (Exception ex)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)